features/sign_up_page.feature

```diff
@@ -0,0 +1,33 @@
+Feature: Sign Up Page
+    As a user
+    In order sign up to CA Mailboxer
+    I want to be able to visit a sign up page and create an account
+
+Scenario: Submit sign up form [Happy path]
+    When I visit the "sign up" page
+    Then I should be able to fill in "Name" with "Camron"
+    And I should be able to fill in "Email" with "camron@heroku.com"
+    And I should be able to fill in "Password" with "camron1234"
+    And I should be able to fill in "Password confirmation" with "camron1234"
+    Then I should see "Log in" link
```

Comment:
Why do we need to see the login link?

Reply:
We thought we had to check to make sure that that link was on the page as well. But guessing we don't. Removed that now.